packages/frontend-shared/cypress/support/mock-graphql/mountFragment.ts

```diff
@@ -48,6 +48,12 @@ export const registerMountFn = ({ plugins }: MountFnOptions = {}) => {
 
       options.global.plugins.push(createI18n())
 
+      options.global.plugins.push({
+        install (app) {
+          app.use(urql, testUrqlClient(context, undefined, mutationResolvers))
+        },
+      })
+
```

Comment:
This allows us to stub mutation resolvers for components that use `mount` rather than `mountFragment`